dist/index.js

```diff
@@ -57,13 +57,8 @@ function koaDevware(compiler, options) {
 
   function middleware(context, next) {
     return new _promise2.default(function (resolve, reject) {
-      // https://github.com/webpack/docs/wiki/plugins#donestats-stats
-      compiler.plugin('done', function (stats) {
-        resolve(stats);
-      });
-
-      compiler.plugin('failed', function (error) {
```

Comment:
Perhaps I shouldn't have removed that one, but still more error-handling is needed. For example the case where a non-existing resource is requested. Then the promise needs to be resolved/rejected.